connector/subgraph/schema.graphql

```diff
@@ -0,0 +1,36 @@
+type Vote @entity {
+  id: ID!
+  orgAddress: Bytes!
+  appAddress: Bytes!
```

Comment:
Could we move this two to the bottom?

Reply:
any reason? ahah is exactly like this in all the other examples and also conviction

Reply:
Just because i think logically fells more intuitive to have the main attributes related to the entity first and everything else after. 
In conviction voting we have them at the bottom actually https://github.com/1Hive/conviction-voting-app/blob/master/connector/subgraph/schema.graphql :)